package.json

```diff
@@ -196,5 +196,8 @@
   "engines": {
     "npm": ">=2.0.0"
   },
-  "typings": "./dist/cjs/Rx.d.ts"
+  "typings": "./dist/cjs/Rx.d.ts",
+  "dependencies": {
+    "symbol-observable": "^0.2.2"
```

Comment:
I've changed this to `dependencies` instead of `devDependencies` since it's required dependency not only dev perspective.